javascript-if/main.js

```diff
@@ -0,0 +1,70 @@
+/* exported isUnderFive,
+            isEven,
+            startsWithJ,
+            isOldEnoughToDrink,
+            isOldEnoughToDrive,
+            isOldEnoughToDrinkAndDrive,
+            categorizeAcidity,
+            introduceWarnerBro
+ */
+
+function isUnderFive(number) {
+  var isUnderFiveResult = number < 5;
+  return isUnderFiveResult;
+}
+
+function isEven(number) {
+  var isEvenResult = (number % 2) === 0;
+  return isEvenResult;
+}
+
+function startsWithJ(string) {
+  var startsWithJResult = string[(0)] === 'J';
+  return startsWithJResult;
+}
+
+function isOldEnoughToDrink(person) {
+  var isOldEnoughToDrinkResult = person.age > 20;
+  return isOldEnoughToDrinkResult;
+}
+
+function isOldEnoughToDrive(person) {
+  var isOldEnoughToDrinkResult = person.age > 15;
+  return isOldEnoughToDrinkResult;
+}
+
+function isOldEnoughToDrinkAndDrive(person) {
+  var isOldEnoughToDrinkAndDriveResult = (isOldEnoughToDrink > 15) === (isOldEnoughToDrive > 20);
+  return isOldEnoughToDrinkAndDriveResult;
+}
+
+function categorizeAcidity(pH) {
+  var categorizeAcidityResult = pH;
+
+  if ((pH >= 0) || (pH <= 6)) {
+    categorizeAcidityResult = 'Acid';
+  }
+  if ((pH >= 8) && (pH <= 14)) {
+    categorizeAcidityResult = 'Base';
+  }
+  if ((pH === 7)) {
+    categorizeAcidityResult = 'neutral';
+  }
+  if ((pH < 0) || (pH > 14)) {
+    categorizeAcidityResult = 'invalid pH level';
+  }
+  return categorizeAcidityResult;
+}
```

Comment:
pH level is a continuous scale from `0` to `14`. Blood (pH of 7.4) and milk (pH of 6.45) should be classified correctly:

---

![image](https://user-images.githubusercontent.com/16143771/169163550-3da00774-ae16-472a-a0c3-06adcc81e250.png)